package.json

```diff
@@ -107,7 +107,7 @@
     "clsx": "2.1.1",
     "date-fns": "3.6.0",
     "electron-log": "5.1.7",
-    "electron-updater": "6.3.2",
+    "electron-updater": "6.2.1",
```

Comment:
Intentional downgrade?

Reply:
Actually no, good catch. Will correct that